features/scrums.feature

```diff
@@ -25,4 +25,9 @@ Feature: Scrums Index
     Then I should see a modal window with the first scrum
     When I close the modal
     And I click the second scrum in the timeline
-    Then I should see a modal window with the second scrum
\ No newline at end of file
+    Then I should see a modal window with the second scrum
+
+  Scenario: Videos with youtube id nill do not show
```

Comment:
That was the aim of this PR earlier but because project page and user profile page (https://www.agileventures.org/projects/websiteone#video) are showing en error so to be consistent we @tansaku @mattlindsey and me decided to display an error message.